food.go

```diff
@@ -246,7 +246,14 @@ func (f *Food) Link(pkg *Package) error {
 	barrelDir := filepath.Join(home.Barrel(), f.Name, f.Version)
 	for _, r := range pkg.Resources {
 		// TODO: run this in parallel
-		destPath := filepath.Join(home.HomePrefix, r.InstallPath)
+
+		// We assume every Food's InstallPath begins with `bin/` (`bin\\` on Windows)
```

Comment:
Is this a right assumption?

I did check all the foods available in https://github.com/fishworks/fish-food/tree/main/Food and found every existing food has `installpath` under `bin/`, which backs my assumption, but not sure if that's by-design.

Reply:
This isn't always a correct assumption. While all of the current packages just install a standalone binary, many could install manpages under `man/` or shared object files under `lib/`.